lib/cib/Makefile.am

```diff
@@ -20,7 +20,7 @@ libcib_la_SOURCES	+= cib_ops.c
 libcib_la_SOURCES	+= cib_remote.c
 libcib_la_SOURCES	+= cib_utils.c
 
-libcib_la_LDFLAGS	= -version-info 32:0:5
+libcib_la_LDFLAGS	= -version-info 33:0:6
```

Comment:
Where are these numbers coming from? I've never touched them

Reply:
We update them via the `maint/bumplibs` script. There are arcane rules for updating them that make sense once you understand them but aren't intuitive. If you're feeling masochistic, see https://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html